### PR TITLE
fix: update supported range of node versions

### PR DESCRIPTION
**Pull request by sibiraj-s**
## I'm submitting a...

```
[x] Bug Fix
[ ] Feature
[ ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [x] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added

## Description

This is to align the supported nodejs versions with [angular/cli v12.0.3](https://github.com/angular/angular-cli/releases/tag/v12.0.3).

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by sibiraj-s** [comment]
I am not sure whether it should be same as angular cli. which has `^12.14.1 || >=14.0.0`, that drops nodejs 13 support here.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "scss"
   ],
   "engines": {
-    "node": ">=12.14.1 <16.0.0"
+    "node": ">=12.14.1"
```
Comment (alan-agius4) [inline package.json:17]: thanks for this 

```suggestion
    "node": “^12.14.1 || >=14.0.0”
```

**Comment by sibiraj-s** [comment]
done @alan-agius4

**Review by alan-agius4 — APPROVED** [review]
Thanks

**Pull request merged**